h/migrations/versions/615358b6c428_move_all_groups_to_default_organization.py

```diff
@@ -0,0 +1,60 @@
+"""
+Move all groups to default organization.
+
+Revision ID: 615358b6c428
+Revises: f052da9df33b
```

Comment:
This comment is out of date. The `down_revision` below is 8a7f31c4525d.

Reply:
Fixed. We should really edit the migration script template to remove these pointless comments that duplicate the IDs